readme.md

```diff
@@ -195,6 +195,13 @@ Default: `10000`
 
 Timeout in milliseconds for getting a response.
 
+##### onProgress
+
+Type: `Function`<br>
+Default: `10000`
```

Comment:
Huh?

Reply:
Oh, just forgot to remove that part :P (quick copy & paste)